app/config/question-bank.js

```diff
@@ -499,30 +514,102 @@ module.exports = {
             {
               key: 'project-items-A1',
               value: 'Mild acidification equipment',
-              hint: ['', ''],
+              hint: {
+                html: `<span>You must buy all 4 of the following items:</span>
+                <ul>
+                  <li>acid storage</li>
+                  <li>dosing equipment</li>
+                  <li>mixing tank</li>
+                  <li>pump</li>
+                </ul>
+                `
+              },
               mustSelect: true,
-              errorMustSelect: 'This option must be selected -- !!'
+              errorMustSelect: 'You must select mild acidification equipment'
             },
             {
               key: 'project-items-A2',
               value: 'Acidification infrastructure',
-              hint: ''
-            },
-            {
-              key: 'project-items-A3',
-              value: 'Slurry pipework'
+              hint: {
+                text: 'Any work to adapt or install pipework, pumps etc to get slurry into the acidification system and then out to storage.'
+              }
             }
           ],
           yarKey: 'projectItems'
 
         },
+        {
+          key: 'slurry-application',
+          order: 81,
+          title: 'Slurry application - title',
```

Comment:
we are adding 10 to each new question, I think you should change all the order numbers, by the way, I think orders do not do anything because we are calling next question with nextUrl inside of each function, I do not know why we are using order in question 🤔

Reply:
don't worry about order for now
we added 10 to each question so if we have new pages they can fit in (up to 9)
so as long as each question order is greater than the previous one - all is fine